modules/ztp-creating-ztp-custom-resources-for-multiple-managed-clusters.adoc

```diff
@@ -0,0 +1,13 @@
+// Module included in the following assemblies:
+//
+// *scalability_and_performance/ztp-zero-touch-provisioning.adoc
+
+[id="ztp-creating-ztp-custom-resources-for-multiple-managed-clusters_{context}"]
+= Creating ZTP custom resources for multiple managed clusters
```

Comment:
@StephenJamesSmith A lot of these changes look like they're ZTP documents with nothing in common with the NBDE implementation guide - Do we have a bad rebase or something going on in here?

Reply:
@lack The files prefixed with "ztp-nbde" are the NBDE files.

Reply:
Two potential issues with that:
- Having them in the same commit as other non-related changes is confusing
- The NBDE information is not part of ZTP. They are orthogonal.